app/driver.go

```diff
@@ -93,6 +93,10 @@ func DeployDriverCmd() cli.Command {
 				EnvVar: EnvCSIProvisionerName,
 				Value:  csi.DefaultCSIProvisionerName,
 			},
+			cli.StringFlag{
```

Comment:
You can shoot this down, it just makes testing easier because I can compile and run locally

Reply:
That's fine. Not a big deal.